hydra-node/test/Test/Util.hs

```diff
@@ -0,0 +1,62 @@
+module Test.Util where
+
+import Cardano.Prelude hiding (SrcLoc, callStack, throwIO)
+import Control.Monad.Class.MonadThrow (MonadThrow (throwIO))
+import Control.Monad.Class.MonadTimer (DiffTime, MonadTimer, timeout)
+import Control.Monad.IOSim (IOSim, runSim)
+import Data.List (isInfixOf)
+import Data.String (String)
+import GHC.Stack (SrcLoc, callStack)
+import Test.HUnit.Lang (FailureReason (ExpectedButGot, Reason), HUnitFailure (HUnitFailure))
+
+failure :: (HasCallStack, MonadThrow m) => String -> m a
```

Comment:
Are we settling for tuple-style or arguments-style constraints? Seems like we are using both in the code base, could be something to put in our coding conventions.

Reply:
If we have to settle on one style, I'd cast a vote for the tuple-style :grimacing: